From a70e74e0eedb436c4126ac69b0beaa4ab848b78e Mon Sep 17 00:00:00 2001
From: Steve Purcell <steve@sanityinc.com>
Date: Mon, 14 Mar 2016 09:18:16 +1300
Subject: Remove autoload for .text

While .md and .markdown are clearly markdown files, the handling of .text is not a choice that markdown-mode should be making on the user's behalf without any opportunity to prevent it.

diff --git a/markdown-mode.el b/markdown-mode.el
index b7a32cc..978e040 100644
--- a/markdown-mode.el
+++ b/markdown-mode.el
@@ -6634,8 +6634,6 @@ BEG and END are the limits of scanned region."
 ;;;###autoload
 (add-to-list 'auto-mode-alist '("\\.markdown\\'" . markdown-mode))
 ;;;###autoload
-(add-to-list 'auto-mode-alist '("\\.text\\'" . markdown-mode))
-;;;###autoload
 (add-to-list 'auto-mode-alist '("\\.md\\'" . markdown-mode))
 
 
-- 
cgit v0.10.2

